Xamarin.Forms.Platform.Android/AppCompat/MasterDetailPageRenderer.cs

```diff
@@ -236,7 +236,8 @@ protected override void Dispose(bool disposing)
 
 				Device.Info.PropertyChanged -= DeviceInfoPropertyChanged;
 
-				RemoveDrawerListener(this);
+				if (!this.IsDisposed())
```

Comment:
While you're at it might as well standardize this dispose method overall

https://github.com/xamarin/Xamarin.Forms/issues/6305

- move _disposed check to first line and just return if true
- set _isdisposed regardless